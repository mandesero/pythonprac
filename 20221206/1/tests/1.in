class C(metaclass=dump):
    def __init__(self, val):
        self.val = val

    def add(self, other, another=None):
        return self.val + other + (another or self.val)

    def boba(self):
        print("Boba")


c = C(10)
print(c.add(9))
print(c.add(9, another=11))
