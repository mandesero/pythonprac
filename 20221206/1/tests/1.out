0_0.2
0_0.3
1_0.2
1_0.3
2_0.2
3_0.2
2_0.3
4_0.2
3_0.3
5_0.2
